lib/components/Input/CheckboxInput.module.scss

```diff
@@ -40,7 +40,7 @@ $outline-focus-border-width: 1px;
 
         &.selected .checkbox-button {
             color: themed('color-text-white');
-            background-color: themed('color-bg-btn-primary-rest');
+            // background-color: themed('color-bg-btn-primary-rest');

```

Comment:
should this be commented out

Reply:
Nothing changes after removing it, and it always overrides theme by mistake.